### PR TITLE
[ch22164]Terminate PD functionality part 1

### PR DESCRIPTION
**Pull request by lajoskovago**
[ch22164]Terminate PD functionality

### CODE REVIEW

File: common/layout/status-buttons.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  _createTerminationDialog() {
+    this._terminationDialog = document.createElement('pd-termination') as PdTermination;
+    document.querySelector('body')!.appendChild(this._terminationDialog);
+
+    this._terminationDialog.terminationElSource = this;
+    this._setStatusTerminated();
+  }
+
+  _setStatusTerminated() {
```
Comment (acory) [inline common/layout/status-buttons.ts:51]: confusing method name given its content

File: common/components/intervention/pd-termination.ts [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+          fireEvent(this, 'toast', {text: 'Can not update intervention'});
+        })
+        .finally(() => {
+          fireEvent(this, 'global-loading', {
+            active: false,
+            loadingSource: 'intervention-actions'
+          });
+        });
+    }
+  }
```
[the remark this answers is not in the document]
Comment (lajoskovago) [reply common/components/intervention/pd-termination.ts:200]: 2. there is an issue with connecting mixins, when the Terminate Button will come from the backend, it wil be fixed

File: common/components/intervention/pd-termination.ts [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  terminationElSource!: LitElement;
+
+  @property({type: Boolean})
+  uploadInProgress = false;
+
+  @property({type: Boolean, reflect: true})
+  dialogOpened!: boolean;
+
+  @property({type: Object})
+  environmentFlags: EnvFlags | null = null;
```
[the remark this answers is not in the document]
Comment (lajoskovago) [reply common/components/intervention/pd-termination.ts:136]: there is an issue with connecting mixins, when the Terminate Button will come from the backend, it wil be fixed

**Review by acory — APPROVED** [review]

**Comment by acory** [comment]
to be continued

**Pull request merged**